README.md

```diff
@@ -48,5 +48,46 @@ By listing the `CSIAddonsNode` CRs, the CSI-Addons Controller knows how to
 connect to the side-cars. By checking the supported capabilities of the
 side-cars, it can decide where to execute operations that the user requested.
 
+### Installation
+
+This project uses `kustomize` and a `Makefile` for deploying. By running the
+command
+
+```console
+$ make deploy
+...
+customresourcedefinition.apiextensions.k8s.io/csiaddonsnodes.csiaddons.openshift.io created
```

Comment:
@nixpanic looks like fencing CRD is missing here

Reply:
Indeed, but it is just a (partial) copy/paste of what happens now. It seems that `config/crd/bases/csiaddons.openshift.io_networkfences.yaml` is not applied automatically. This should be addressed in a different PR.

Reply:
reported this as #50.